source/extensions/filters/http/dynamic_forward_proxy/proxy_filter.cc

```diff
@@ -71,19 +71,9 @@ Http::FilterHeadersStatus ProxyFilter::decodeHeaders(Http::RequestHeaderMap& hea
     return Http::FilterHeadersStatus::Continue;
   }
 
-  const bool should_use_dns_cache_circuit_breakers =
-      Runtime::runtimeFeatureEnabled("envoy.reloadable_features.enable_dns_cache_circuit_breakers");
-
-  circuit_breaker_ = config_->cache().canCreateDnsRequest(
-      !should_use_dns_cache_circuit_breakers
-          ? absl::make_optional(std::reference_wrapper<ResourceLimit>(
-                cluster_info_->resourceManager(route_entry->priority()).pendingRequests()))
-          : absl::nullopt);
+  circuit_breaker_ = config_->cache().canCreateDnsRequest();
```

Comment:
drive-by, can you remove the un-needed runtime_features include and dep please?

Reply:
fixed